src/python/pants/engine/isolated_process.py

```diff
@@ -69,13 +69,20 @@ def __new__(
     )
 
 
-class ExecuteProcessResult(datatype(['stdout', 'stderr', 'output_directory_digest'])):
+class ExecuteProcessResult(datatype([('stdout', binary_type),
```

Comment:
I have typically been indenting these as follows when it's too long to fit on one line:
```python
class ExecuteProcessResult(datatype([
  ('stdout', binary_type),
  ('stderr', binary_type),
  ('output_directory_digest', DirectoryDigest),
])):
  # ...
```

I have zero problem changing my style, just would be neat to settle on one (I hadn't considered this layout before, it seems perfectly legit).

Reply:
I agree actually! I like that style more and that is how we do things at Foursquare.

In the interest of landing this, can I fix in a follow up PR?